config/global.php

```diff
@@ -72,4 +72,16 @@
                 'tld' => false,
             ))));
     },
+
+    'tracker.request.processors' => array(),
```

Comment:
Also can we avoid that a plugin developer has to add it here? How can we make it super simple for a plugin developer?